pkg/rhttp/datatx/manager/tus/tus.go

```diff
@@ -99,6 +100,7 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 	config := tusd.Config{
 		StoreComposer:         composer,
 		NotifyCompleteUploads: true,
+		Logger:                log.New(appctx.GetLogger(context.Background()), "", 0),
```

Comment:
hmm, now that the discussion is on, let me throw in my 2 cents. Can we pass in the active context instead of creating it on the fly because the active context holds the requestID.

Reply:
There is no active context. This happens on service init